### PR TITLE
Convert Table to List Groups

### PR DESCRIPTION
**Pull request by aldoportillo**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,13 +3,17 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Home</title>
+    <title>Playlist</title>
```
Comment (heratyian) [inline index.html:6]: nice

File: index.html [hunk 2]
```diff
@@ -24,47 +28,43 @@ <h1 class="display-4">Playlist</h1>
...
+            <a href="https://github.com/JoseMDO" target="_blank">
+              Jose Montes De Oca Morfin
             </a>
-          </td>
-        </tr>
-      </tbody>
-    </table>
+          </p>
+        </div>
+        <a href="https://www.youtube.com/watch?v=PrCG4EK1Ehs" target="_blank">
```
Comment (heratyian) [inline index.html:64]: this button is really small. maybe we can use a chevron? https://fontawesome.com/icons/chevron-right?f=classic&s=solid

File: index.html [hunk 3]
```diff
@@ -24,47 +28,43 @@ <h1 class="display-4">Playlist</h1>
...
-          <td>Friends in Low Places</td>
-          <td>
-            <a
-              href="https://www.youtube.com/watch?v=ZcQeocF56Ck"
-              target="_blank"
-              class="btn btn-primary"
-            >
-              Link
+
+    <div class="list-group">
```
Comment (heratyian) [inline index.html:33]: Maybe we can make each cohort a card with a list group? Like in the dashboard.

Comment (heratyian) [reply index.html:33]: https://getbootstrap.com/docs/5.3/components/list-group/#flush

File: index.html [hunk 4]
```diff
@@ -24,47 +28,43 @@ <h1 class="display-4">Playlist</h1>
...
+      <div
```
[the remark this answers is not in the document]
Comment (heratyian) [reply index.html:34]:
```
<a class="list-group-item" href="https://www.youtube.com/watch?v=ZcQeocF56Ck">
  <div>
    artist
    song
    submitted by
  </div>
</a>
```

this has a nice hover state https://getbootstrap.com/docs/5.3/components/list-group/#custom-content

**Review by heratyian — COMMENTED** [review]
This is coming together nicely.

File: index.html (hunk 2 above)
Comment (heratyian) [reply index.html:64]: might not be necessary if we use this https://getbootstrap.com/docs/5.3/components/list-group/#custom-content

**Comment by aldoportillo** [comment]
@heratyian If we make the div clickable we lose the ability to click on the person's github profile. Is that ok?

**Comment by heratyian** [comment]
> @heratyian If we make the div clickable we lose the ability to click on the person's github profile. Is that ok?

yes. seems like a reasonable tradeoff.

**Comment by aldoportillo** [comment]
@heratyian Updated!

**Review by heratyian — APPROVED** [review]
👍

**Review by JoseMDO — APPROVED** [review]
Looks awesome!

**Pull request merged**